src/CodeComb.vNextChina.Hub/EFNodeProvider.cs

```diff
@@ -16,7 +16,7 @@ public EFNodeProvider(TContext db)
         {
             DB = db;
             Nodes = new List<Node>();
-            foreach (var x in DB.Nodes.ToList())
+            foreach (var x in DB.Nodes)

```

Comment:
.ToList will makes the records into memory, so you cannot remove the tolist here.